docs/topics/multiplatform/multiplatform-dsl-reference.md

```diff
@@ -613,7 +613,7 @@ Configurations of source sets are stored inside the corresponding blocks of `sou
 |`resources.srcDir`| Location of resources inside the source set directory.                                                                         |
 |`dependsOn`| [Connection with another source set](multiplatform-share-on-platforms.md#configure-the-hierarchical-structure-manually).       |
 |`dependencies`| [Dependencies](#dependencies) of the source set.                                                                               |
-|`languageSettings`| [Language settings](multiplatform-dsl-reference.md#language-settings) applied to the source set. |
+|`languageSettings`| [Language settings](#language-settings) applied to the source set. |
```

Comment:
JFYI
it's a link to same page, so I removed the `multiplatform-dsl-reference.md` part